# 卢瑟福模型与电子坠落

## 背景

### 卢瑟福模型

卢瑟福模型将原子描述为一个微小的太阳系：正电荷集中在中心的原子核，电子则像行星一样绕核旋转。

为了简化计算，我们假设电子绕核做匀速圆周运动。

以氢原子为例，电子电荷量为 $-e$，质子电荷量为 $+e$，电子质量为 $m_e$，质子质量为 $m_p$，电子绕核做匀速圆周运动的半径为 $r$。

根据牛顿第二定律，电子和质子之间的库仑力提供向心力：

$$
F = \frac{1}{4\pi \epsilon_0} \frac{e^2}{r^2} = m_e \frac{v^2}{r}
$$

电子线速度 $v$ 为：

$$
v = \sqrt{\frac{e^2}{4\pi \epsilon_0 m_e r}}
$$

根据：

$$
m_e a = m_e \frac{v^2}{r}
$$

电子加速度 $a$ 为：

$$
a = \frac{v^2}{r} = \frac{e^2}{4\pi \epsilon_0 m_e r^2} \tag{1}
$$

电子动能 $K$ 为：

$$
K = \frac{1}{2} m_e v^2 = \frac{e^2}{8\pi \epsilon_0 r}
$$

电子势能 $U$ 为：

$$
U = -\frac{1}{4\pi \epsilon_0} \frac{e^2}{r}
$$

电子机械能 $E$ 为：

$$
E = K + U = \frac{e^2}{8\pi \epsilon_0 r} - \frac{e^2}{4\pi \epsilon_0 r} = -\frac{e^2}{8\pi \epsilon_0 r} \tag{2}
$$

### 拉莫尔公式

Larmor Formula

一个加速度为 $a$ 的带电粒子，其辐射功率 $P$ 遵循拉莫尔公式：

$$
P = \frac{2}{3} \frac{q^2 a^2}{4 \pi \epsilon_0 c^3} \tag{3}
$$

> 属于电动力学内容。

## 思想实验

拉莫尔公式告诉我们，电子在加速运动时会辐射能量，因此电子会逐渐失去能量，最终坠落到原子核上。教科书还告诉我们，这一过程非常短暂，约在 $10^{-10}$ 至 $10^{-11}$  秒内完成。

我试图计算出电子坠落到原子核上所需的时间。

首先我将通过符号推导所需时间的数学表达式，然后代入数值计算。

### 符号推导

辐射功率 $P$ 代表单位时间内电子损失的能量。因此，能量的变化率可以写为：

$$
\frac{dE}{dt} = -P
$$

通过链式求导法则，可知：

$$
\frac{dE}{dt} = \frac{dE}{dr} \frac{dr}{dt}
$$

根据 (2) 式，求 $E$ 对 $r$ 的导数：

$$
\frac{dE}{dr} = \frac{d}{dr} \left( - \frac{e^2}{8\pi \epsilon_0 r} \right) = \frac{e^2}{8\pi \epsilon_0 r^2}
$$

回到 $E$ 的链式求导，我们现在有：

$$
-P = \frac{e^2}{8\pi \epsilon_0 r^2} \frac{dr}{dt}
$$

代入拉莫尔公式 (3)，可得：

$$
- \frac{2}{3} \frac{e^2 a^2}{4 \pi \epsilon_0 c^3} = \frac{e^2}{8\pi \epsilon_0 r^2} \frac{dr}{dt}
$$

化简得：

$$
\frac{dr}{dt} = - \frac{e^2 a^2}{6 \pi \epsilon_0 c^3} \frac{8\pi \epsilon_0 r^2}{e^2}
= -\frac{4 r^2}{3 c^3} a^2
$$

代入 $a$ 的表达式 (1)，可得：

$$
\frac{dr}{dt} = -\frac{4 r^2}{3 c^3} \left( \frac{e^2}{4\pi \epsilon_0 m_e r^2} \right)^2
= -\frac{e^4}{12 \pi^2 \epsilon_0^2 m_e^2 c^3 r^2}
$$

对上式分离变量：

$$
r^2 dr = -\frac{e^4}{12 \pi^2 \epsilon_0^2 m_e^2 c^3} dt
$$

两边积分。左边，$r$ 从初始半径 $r_0$ 变为 $0$；右边，$t$ 从 $0$ 变为 $\tau$：

\begin{aligned}
\int_{r_0}^{0} r^2 dr &= -\frac{e^4}{12 \pi^2 \epsilon_0^2 m_e^2 c^3} \int_0^{\tau} dt \\[8pt]
\left. \frac{r^3}{3} \right|_{r_0}^{0} = -\frac{r_0^3}{3} &= -\frac{e^4}{12 \pi^2 \epsilon_0^2 m_e^2 c^3} \tau
\end{aligned}

因此，所需时间 $\tau$ 为：

$$
\tau = \frac{r_0^3}{3} \frac{12 \pi^2 \epsilon_0^2 m_e^2 c^3}{e^4} = \frac{4 \pi^2 \epsilon_0^2 m_e^2 c^3 r_0^3}{e^4}
$$

$$
\boxed{
\tau = \frac{4 \pi^2 \epsilon_0^2 m_e^2 c^3 r_0^3}{e^4}
}
$$

### 数值计算

In [5]:
#| code-fold: false

import numpy as np
from scipy.constants import e, m_e, c, epsilon_0, physical_constants

# 获取波尔半径 (r0)
r0 = physical_constants['Bohr radius'][0]

# 计算所需时间 (tau)
tau = (4 * np.pi**2 * epsilon_0**2 * m_e**2 * c**3 * r0**3) / (e**4)

print("-" * 30)
print(f"常数检查:")
print(f"e         = {e:.4e} C")
print(f"m_e       = {m_e:.4e} kg")
print(f"epsilon_0 = {epsilon_0:.4e} F/m")
print(f"r0        = {r0:.4e} m")
print("-" * 30)
print(f"理论坠毁时间 tau = {tau:.4e} s")
print(f"换算为皮秒: {tau * 1e12:.3g} ps")
print("-" * 30)

------------------------------
常数检查:
e         = 1.6022e-19 C
m_e       = 9.1094e-31 kg
epsilon_0 = 8.8542e-12 F/m
r0        = 5.2918e-11 m
------------------------------
理论坠毁时间 tau = 1.5562e-11 s
换算为皮秒: 15.6 ps
------------------------------


## 总结

通过上述的符号推导与数值计算，我们得出结论：如果按照经典电动力学（卢瑟福模型结合拉莫尔公式），一个氢原子的电子从玻尔半径位置开始，因辐射能量而坠落到原子核表面，仅需：

$$
\tau \approx 1.56 \times 10^{-11} \, \text{s}
$$

即大约 15.6 皮秒。

这个计算结果揭示了经典物理学在微观领域的重大困境，通常被称为“原子塌缩”问题。$10^{-11}$ 秒是一个极短的时间瞬间。这意味着，如果世界遵循纯粹的经典定律，所有的原子都会在瞬间自行毁灭，物质世界根本无法稳定存在。

我们不得不得出结论：卢瑟福原子模型与经典物理学在描述原子结构时存在根本性的缺陷。人们对于原子结构的认识需要引入量子力学等新的物理学理论。